docs/site/index.md

```diff
@@ -8,10 +8,6 @@ permalink: /doc/en/lb4/index.html
 summary: LoopBack is a platform for building APIs and microservices in Node.js
 ---
 
-{% include important.html content="LoopBack 4 is the next step in the evolution
-of LoopBack. It is still in early development and is not yet released.
-" %}
-
```

Comment:
Instead of deleting this note, I am proposing to replace it with a note announcing LB 4.0 GA, as we did elsewhere - see e.g. https://github.com/strongloop/loopback.io/pull/746

```markdown
{% include note.html content="
LoopBack 4 GA (General Availability) has been released in October 2018,
read more in <a href="http://strongloop.com/strongblog/loopback-4-ga">the announcement post</a>
" %}
```